### PR TITLE
default built to require windows 10

### PR DESCRIPTION
**Pull request by xavier2k6**

### CODE REVIEW

File: src/drive_info.cpp [hunk 1]
```diff
@@ -243,7 +243,7 @@ drive_info get_drive_info(std::string const& path)
 		if (dev.handle() == INVALID_HANDLE_VALUE)
 			continue;
 
-#if _WIN32_WINNT >= 0x601
```
Comment (arvidn) [inline src/drive_info.cpp:246]: I don't understand this change. Is there anything in this block that requires windows 10?

**Review by arvidn — APPROVED** [review]

**Comment by xavier2k6** [comment]
BTW, Boost are starting to `default` their target API's to Windows 10 as of [Boost 1.84.0](https://www.boost.org/users/history/version_1_84_0.html)

**Comment by arvidn** [comment]
it appears that having the build system define `_WIN32_WINNT` will make boost.asio think it's building on windows, and try to include `<winapifamily.h>`.

**Comment by arvidn** [comment]
here's my attempt: https://github.com/arvidn/libtorrent/pull/7662

**Pull request closed without merging**